lib/asn1/krb5.asn1

```diff
@@ -223,8 +223,9 @@ AUTHDATA-TYPE ::= INTEGER {
 	KRB5-AUTHDATA-BEARER-TOKEN-JWT(581),  -- JWT token
 	KRB5-AUTHDATA-BEARER-TOKEN-SAML(582), -- SAML token
 	KRB5-AUTHDATA-BEARER-TOKEN-OIDC(583), -- OIDC token
-	KRB5-AUTHDATA-CSR-AUTHORIZED(584),     -- Proxy has authorized client
+	KRB5-AUTHDATA-CSR-AUTHORIZED(584),    -- Proxy has authorized client
                                               -- to requested exts in CSR
+	KRB5-AUTHDATA-SYNTHETIC-CLIENT(585),  -- client synthesized by KDC
 	KRB5-AUTHDATA-GSS-COMPOSITE-NAME(655) -- gss_export_name_composite
```

Comment:
What's the plan  for getting registered authdata type values for these "DO NOT USE in production yet!" uses?

Reply:
I can't speak to @nicowilliams' types, but I'm working on an update to draft-perez-krb-wg-gss-preauth.

Reply:
Oh sorry, this was re: SYNTHETIC-CLIENT. I don't think this one needs to be documented because its only consumer is the Heimdal KDC. But, we do need to check there's a private use number we can safely use.

Reply:
Sorry, I should have been more clear about the question (but you figured it out, fortunately).
I think these are not in an IANA registry, though, but rather something that tlyu had maintained for a long time and maybe has transitioned to ghudson.  https://datatracker.ietf.org/doc/html/draft-ietf-kitten-kerberos-iana-registries-04 stalled at the IETF for lack of energy but suggested that negative values would be suitable for local use (which meshes with some fragmentary memories I have of some negative values in a different codepoint space seeing common use because Microsoft caused them to be de facto standards).